pom.xml

```diff
@@ -211,9 +211,8 @@
         <version.org.wildfly.openssl.wildfly-openssl-solaris-x86_64>${version.org.wildfly.openssl.natives}</version.org.wildfly.openssl.wildfly-openssl-solaris-x86_64>
         <version.org.wildfly.openssl.wildfly-openssl-windows-i386>${version.org.wildfly.openssl.natives}</version.org.wildfly.openssl.wildfly-openssl-windows-i386>
         <version.org.wildfly.openssl.wildfly-openssl-windows-x86_64>${version.org.wildfly.openssl.natives}</version.org.wildfly.openssl.wildfly-openssl-windows-x86_64>
-        <version.org.wildfly.security.elytron>1.8.0.Final</version.org.wildfly.security.elytron>
+        <version.org.wildfly.security.elytron>1.9.0.CR2</version.org.wildfly.security.elytron>
         <version.org.wildfly.security.elytron-web>1.4.0.Final</version.org.wildfly.security.elytron-web>
-        <version.org.wildfly.security.elytron.tool>1.6.0.Final</version.org.wildfly.security.elytron.tool>
```

Comment:
does that mean that Elytron tool has been "put back" with other Elytron modules and will now be updated with the rest of Elytron components?

Reply:
Yes Elytron Tool is now tagged when we tag Elytron so it is no longer required that we tag it separately so it will always come in via the same component upgrade.